sections/experiments/datasets.tex

```diff
@@ -134,106 +141,73 @@ \subsection{Evaluation protocols}\label{sec:evaluationmetrics}
     NDCG = \frac{DCG}{DCG*}
 \end{equation}
 \\\\
-Generally, the papers examined in \autoref{tab:papermetricsprotocols} employ cross validation across a number of folds, in a range of 3-10 folds.
-\cite{GameTheoretic}, \cite{Soft-RoughArticle}, \cite{SinghUserItem}, \cite{StackedRecurrentNeuralPaper} and \cite{ContextualInfluencePaper} do not specify a number of folds, but rather just split the data into either two or three sets, with 20\%-80\% being used for training, and the remaining either spread across validation and testing or just testing.
-\cite{AuxiliaryInformationPaper} and \cite{ArtificalBeePaper} do not specify their protocol, opting to simply report results.
+The papers examined throughout this research employ cross validation across a number of folds, in a range of 5-10 folds.
+Most of them comform to splitting the dato into an 80\%/20\% split of training and test data, as seen on \autoref{tab:methodsused}.
+Another interesting aspect to look into for the various papers that has been researched is which metrics they use for comparisons, the results of this can be seen on \autoref{tab:metricsused}.
```

Comment:
```suggestion
Another interesting aspect to look into for the various papers that have been researched is which metrics they use for comparisons.
The results of this can be seen on \autoref{tab:metricsused}.
```